gtda/mapper/visualization.py

```diff
@@ -39,7 +37,7 @@ def plot_static_mapper_graph(
     Two clusters from different pullback cover sets can overlap; if they do, an
     edge between the corresponding nodes in the graph may be drawn.
 
-    Nodes are colored according to `color_variable` and `node_color_statistic`
+    Nodes are colored according to `color_features` and `node_color_statistic`
```

Comment:
What about `color_data`?

Reply:
That would be incorrect if e.g. `color_features` selects a subset of columns from `color_data`.  One could say that the colors are determined by `color_features` *applied to* `color_data`, but I'm not sure saying it like this would help in a first read.

Reply:
You're right, it wouldn't be correct either. Simpler is better, let's keep it as is

Reply:
Ok!